cclient/request/find_transactions.h

```diff
@@ -19,40 +19,31 @@ typedef struct {
    * List of addresses.
    * Transactions with any of these addresses as input/output will be returned
    */
-  flex_hash_array_t* addresses;
+  hash243_queue_t addresses;
```

Comment:
Do we need ordering here ? A stack would be more efficient.

Reply:
why not? It's more flex.